platform/admin/configurator/admin-configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/ConfigOperation.java

```diff
@@ -34,58 +45,244 @@
  */
 public abstract class ConfigOperation implements Operation<Void> {
```

Comment:
@coyotesqrl let me know if there's anything I can do to improve upon these impls. They are very similar to their property counterparts but they operate on Dictionaries.